example-crates/example_electrum/src/main.rs

```diff
@@ -60,19 +60,22 @@ pub struct ScanOptions {
     pub batch_size: usize,
 }
 
-type ChangeSet = WalletChangeSet<Keychain, ConfirmationHeightAnchor>;
+type ChangeSet = (
+    local_chain::ChangeSet,
+    indexed_tx_graph::ChangeSet<ConfirmationHeightAnchor, keychain::ChangeSet<Keychain>>,
+);
 
 fn main() -> anyhow::Result<()> {
     let (args, keymap, index, db, init_changeset) =
```

Comment:
:bulb: deconstruct `init_changeset` into `(disk_local_chain, disk_tx_graph)` to avoid using `.1` and `.0` below.